requirements/requirements_multimodal.txt

```diff
@@ -0,0 +1,20 @@
+addict
+basicsr
```

Comment:
This is compiled under some extras - do we need those ? If so,. Can it be made optional ? Under "multimodal_extras"